### ISSUE TITLE
Windows runtime terminating in an unusual way

### ISSUE THREAD

**Issue by eagles051387** [LMMS/lmms#512]
every time i try to open the lmms its stop and tells me (thisapplication has requested the runtime to terminate it in and unusual way ) and than closes the program some one help me please i look on youtube and cant find a video for my problem only video on how to use it

**Comment by BeaQueen** [comment]
I tried all the solutions shared on this post and LMMS (version 1.1.3) is still not running on my computer {Windows 10 64bits}.
[Intel i7-6700K 4.GHz, 16RAM GB, Sound card Approx APPPCI71 PCI 7.1]  
![lmms](https://cloud.githubusercontent.com/assets/16983449/13492440/511f7d20-e138-11e5-935a-833b809b06cf.png)

**Comment by musikBear** [comment]
@BeaQueen This thread is for a general crash when LMMS is closed *down* fully normally with a click in the red-x, or the file context-menu.
If i understand you correctly, you cant run LMMS at all?
That is different, and does not belong *here*
''I dont have an emediate 'fix' for your issue. Others uses win10 successfully, but try
http://lmms.io/forum/viewforum.php?f=7

**Comment by tresf** [comment]
> @BeaQueen This thread is for a general crash when LMMS is closed down fully normally with a click in the red-x, or the file context-menu.

No it's not, it's identical to her problem.

> That is different, and does not belong here

It's OK here.  Although it's a known issue, it's a quite severe one and @BeaQueen has at least posted to the topic versus creating a new duplicate.

@BeaQueen have you tried to reinstall your sound card drivers?  I've have a computer start working simply by uninstalling the sound card driver and getting a new driver from Windows.  In other cases the opposite has helped.

You may also want to move your `%USERPROFILE%/.lmmsrc.xml` file to see if you can force a fallback on to SDL.  Alternatively you can try to copy a config file from another computer with `DummyAudio` selected and see if that will help it load.

@musikBear is correct in the sense that you'll get more end-user help on the forums.  Most of the devs don't run Windows at all, and those that do, can't reproduce the problem, making this bug very difficult to debug from a development perspective, leaving the forums a better home for discussion.

**Comment by musikBear** [comment]
Why dig in the DB to recreate my mistake. I realized i was on the wrong track emediately and removed the comment?
@BeaQueen  If you already has tried as you wrote
> I tried all the solutions shared on this post

Then the driver thing is also old news.
One way to get a working LMMS could perhaps be to try the 32x windows version, or perhaps this
https://lmms.io/forum/viewtopic.php?t=392&start=10#p15566
Is what you need?

**Comment by BeaQueen** [comment]
Well, I'll tell you what happened.
First, I uninstalled all my sound card drivers, LMMS worked but it didn have any sound.
I installed all my sound card drivers, my headphones and my speakers didn't work but LMMS worked.
I configurated all the drivers software, LMMS stopped working again with the Runtime Error again but all my sound stuff was working again. 
I downloaded Bitwig and replaced the portaudio for libportaudio. Error 0xc000007b ocurred. I tried it in different ways and nothing worked. I surrender
Thank you all for the info

**Comment by Umcaruje** [comment]
@BeaQueen could you post your `.lmmsrc.xml` (its located in your user folder)

You can use a service like https://gist.github.com/ or http://pastebin.com/

**Comment by BenMcLean** [comment]
Seeing as how this is a serious bug (crash on launch!) that is affecting lots of people making the software completely unusable, why does this issue say closed? Is there an open version anywhere I should be following?

I want to switch to using more free software for my music production and maybe eventually migrate to Linux completely, but I'm stuck using commercial DAW for now because of this.

**Comment by tresf** [comment]
> Seeing as how this is a serious bug (crash on launch!) that is affecting lots of people making the software completely unusable on current gen systems, why does this issue say closed? Is there an open version anywhere I should be following?

The issue was resolved here: https://github.com/LMMS/lmms/issues/512#issuecomment-48825153

You're absolutely correct, this is still a bug, but it's not necessarily one we can easily fix.  The `portaudiosharp` library is no longer maintained; built with .NET (Mono) and needs review to see if it's actually GPL2 compliant.  Whether this bug is open or closed, we keep this crash as part of our known issues; it's not forgotten.

The underlying problem is most likely an upstream bug with the portaudio library itself and proper resolution should be made with the portaudio team.  Since we don't have any active Windows developers, this won't change for a while.

> I want to switch to using more free software for my music production and maybe eventually migrate to Linux completely, but I'm stuck using commercial DAW for now because of this.

This won't happen on Linux and I strongly encourage you to try the aforementioned workaround.  Note, this doesn't happen on all computers, only some.  I believe the issue to be caused by specific device drivers as I've successfully fixed the crash by downgrading and/or upgrading sound card device drivers, which should be attempted as well.

**Comment by musikBear** [comment]
@BenMcLean Could you post soundcard and driver information. All information regarding this issue, could eventually be useful.

**Comment by BenMcLean** [comment]
I have RealTek High Definition Audio in my ASUS Q551LN-BBI706 laptop.

If you want me to copy-paste detailed info then let me know what tool to use to get you that info. Anything I can do to get this issue fixed.

**Comment by kifujin** [comment]
It's been a while, but I think I eventually found a DLL in system32 that it
was loading instead of the one in the directory for some reason, so I got
the 64 bit version working by uninstalling the thing that was providing
that.

My memory may be flaky, but I think that's what happened...

**Comment by tresf** [comment]
> It's been a while, but I think I eventually found a DLL in system32 that it
was loading instead of the one in the directory for some reason, so I got
the 64 bit version working by uninstalling the thing that was providing
that.

That's what I've experienced as well.  This usually comes down to a drivers.  That doesn't excuse the crash, but in the one time I've experienced this it was resolved by using Device Manager to remove the sound card using the "Remove driver software" checkbox.   Once the driver was reinstalled through Windows update, the problem was resolved.

This technique hasn't work for all people though.  There's most certainly a bug with the backend we use however we lack the proper Windows environment to reproduce and then to debug.
